### ISSUE TITLE
[L5.5] Asset loading in CMS fails 

### ISSUE THREAD

**Issue by Teranode** [octobercms/october#2938]
![](https://image.prntscr.com/image/hi2GF8scQWyIXb7lj-SaXQ.png)

Upon trying to load an asset via CMS I get that error:

heres the log:

```
[2017-06-22 12:57:37] production.ERROR: BadMethodCallException: Call to undefined method Cms\Classes\Asset::fireEvent() in /home/animezero/public_html/vendor/october/rain/src/Extension/ExtendableTrait.php:395
Stack trace:
#0 /home/animezero/public_html/vendor/october/rain/src/Extension/Extendable.php(42): October\Rain\Extension\Extendable->extendableCall('fireEvent', Array)
#1 /home/animezero/public_html/modules/backend/widgets/Form.php(1025): October\Rain\Extension\Extendable->__call('fireEvent', Array)
#2 /home/animezero/public_html/modules/backend/widgets/Form.php(307): Backend\Widgets\Form->applyFiltersFromModel()
#3 /home/animezero/public_html/modules/backend/widgets/Form.php(216): Backend\Widgets\Form->prepareVars()
#4 /home/animezero/public_html/modules/cms/controllers/index/_form_page.htm(7): Backend\Widgets\Form->render()
#5 /home/animezero/public_html/modules/system/traits/ViewMaker.php(243): include('/home/animezero...')
#6 /home/animezero/public_html/modules/system/traits/ViewMaker.php(97): Backend\Classes\Controller->makeFileContents('/home/animezero...', Array)
#7 /home/animezero/public_html/modules/cms/controllers/Index.php(134): Backend\Classes\Controller->makePartial('_form_page.htm', Array)
#8 [internal function]: Cms\Controllers\Index->index_onOpenTemplate()
#9 /home/animezero/public_html/modules/backend/classes/Controller.php(532): call_user_func_array(Array, Array)
#10 /home/animezero/public_html/modules/backend/classes/Controller.php(423): Backend\Classes\Controller->runAjaxHandler('onOpenTemplate')
#11 /home/animezero/public_html/modules/backend/classes/Controller.php(233): Backend\Classes\Controller->execAjaxHandlers()
#12 /home/animezero/public_html/modules/backend/classes/BackendController.php(95): Backend\Classes\Controller->run('index', Array)
#13 [internal function]: Backend\Classes\BackendController->run('cms')
#14 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): call_user_func_array(Array, Array)
#15 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(44): Illuminate\Routing\Controller->callAction('run', Array)
#16 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Route.php(203): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Backend\Classes\BackendController), 'run')
#17 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Route.php(160): Illuminate\Routing\Route->runController()
#18 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(582): Illuminate\Routing\Route->run()
#19 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(30): Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(41): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Routing\Middleware\SubstituteBindings->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#24 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#25 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#26 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(64): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#27 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#28 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#30 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(59): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#33 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#34 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#36 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(584): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#37 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(543): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#38 /home/animezero/public_html/vendor/october/rain/src/Router/CoreRouter.php(20): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#39 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(174): October\Rain\Router\CoreRouter->dispatch(Object(Illuminate\Http\Request))
#40 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(30): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#41 /home/animezero/public_html/plugins/teranode/proxy/middleware/RouteMiddleware.php(55): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#42 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Teranode\Proxy\Middleware\RouteMiddleware->handle(Object(Illuminate\Http\Request), Object(Closure))
#43 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(46): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#45 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#46 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#47 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#48 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(149): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#49 /home/animezero/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(116): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#50 /home/animezero/public_html/index.php(43): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#51 {main}  
```

**Comment by Teranode** [comment]
it relates to this commit:
https://github.com/octobercms/october/commit/9eb0cc24aefef2ffcd54c08dba0a640d6d602106

**Comment by daftspunk** [comment]
Thanks! Fixed by c3570ffd7e8004f93e8cc5f4349d02e890b67490
